modules/Core/includes/endpoints/UserInfoEndpoint.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @param int $id NamelessMC ID of user to view
+ * @param string $username The NamelessMC username of the user to view
+ * @param string $uuid The Minecraft UUID of the user
+ * 
+ * @return string JSON Array
+ */
+class UserInfoEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'userInfo';
+        $this->_module = 'Core';
+        $this->_description = 'Get information about a NamelessMC user';
+    }
+    
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.